src/js/tester.js

```diff
@@ -38,10 +39,12 @@
 					type: 'POST',
 					url : 'https://fcm.googleapis.com/fcm/send',
 					contentType : 'application/json',
-					data : JSON.stringify(msg),
-					dataType : 'json',
-					succes : function(data) {
-						console.log(data);
+					data : JSON.stringify(notification),
+					success : function(data) {
+						console.log("Success!");
```

Comment:
Some thing like [this](https://www.webcomponents.org/element/PolymerElements/paper-toast/paper-toast) could be nice for letting user know the notification was sent?